native/cocos/platform/ohos/libcocos/src/main/java/com/cocos/lib/CanvasRenderingContext2DImpl.java

```diff
@@ -306,6 +306,22 @@ public void setLineJoin(String lineJoin) {
         mLineJoin = lineJoin;
     }
 
+    public void setShadowBlur(float blur) {
+        //
+    }
+
+    public void setShadowColor(float r, float g, float b, float a) {
+        //
+    }
+
+    public void setShadowOffsetX(float offsetX) {
+        //
+    }
+
+    public void setShadowOffsetY(float offsetY) {
+        //
+    }
```

Comment:
Why need these methods as there is not implementation.

Reply:
ohos and android use the same jni implement

Reply:
ok